### PR TITLE
Added meilisearch

### PR DESCRIPTION
**Pull request by Ak4zh**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/logos/meilisearch.png [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
```
Comment (LiquidITGuy) [inline public/v4/logos/meilisearch.png:1]: I'm not sure that is a png file ;)

Comment (LiquidITGuy) [inline public/v4/logos/meilisearch.png:1]: I'm not sure that is a png file ;)

File: public/v4/apps/meilisearch.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+services:
+    $$cap_appname:
+        image: getmeili/meilisearch:$$cap_app_version
+        volumes:
+            - $$cap_appname-data-ms:/data.ms
+        restart: always
+        caproverExtra:
+            containerHttpPort: '7700'
+caproverOneClickApp:
+    variables:
```
Comment (LiquidITGuy) [inline public/v4/apps/meilisearch.yml:11]: do we need the MEILI_MASTER_KEY to connect ?

**Review by LiquidITGuy — COMMENTED** [review]
I cannot wait to try it so I tried it and it works :) 

thank you (this tool was on my todo for my server :) )

File: public/v4/apps/meilisearch.yml (hunk 2 above)
Comment (Ak4zh) [reply public/v4/apps/meilisearch.yml:11]: I am working on adding all the environment variables

File: public/v4/apps/meilisearch.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: getmeili/meilisearch:$$cap_app_version
+        volumes:
+            - $$cap_appname-data:/$$cap_db_path
+            - $$cap_appname-snapshot:/$$cap_snapshot_dir
+            - $$cap_appname-dump:/$$cap_dump_dir
```
Comment (githubsaturn) [inline public/v4/apps/meilisearch.yml:8]: Defining the "inside container" paths as customizable variables usually provide little value to the user. 

Users don't really care about what path is used inside the container. Let's remove these and just hardcode these:
- `$$cap_db_path`
- `$$cap_snapshot_dir`
- `$$cap_dump_dir`

**Review by githubsaturn — COMMENTED** [review]
Since @LiquidITGuy  is testing this, I'll leave it up to him to do a final review.

@LiquidITGuy  - let me know once this looks okay and I'll merge it. Thanks!

File: public/v4/apps/meilisearch.yml (hunk 3 above)
Comment (Ak4zh) [reply public/v4/apps/meilisearch.yml]: Thanks for the review.
Committed the required changes.

**Comment by githubsaturn** [comment]
@LiquidITGuy - does this look good to you?

**Comment by githubsaturn** [comment]
Assuming things are okay, merging!

**Pull request merged**